src/modules/70-pipeline/strings/strings.en.yaml

```
@@ -414,6 +414,7 @@ execution:
     pauseStage: Pause Stage
     abortStage: Abort Stage
     downloadLogs: Download logs
+    viewCompiledYaml: View Compiled Yaml
```

Looks like overall we use `YAML` instead of `Yaml`
```suggestion
    viewCompiledYaml: View Compiled YAML
```